vignettes/detailed-canvas-export-info.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Detailed Canvas Export Info"
+output: rmarkdown::html_vignette
+vignette: >
```

Comment:
This strikes me as important documentation but categorically different from the other vignettes. Perhaps put it under a distinct heading?
<img width="1610" alt="Screenshot 2024-11-13 at 8 11 51 AM" src="https://github.com/user-attachments/assets/5db79126-3089-4fe9-aa7f-90a390af06a8">

Reply:
^^ I second that, this could be a good time to organize the vignettes, especially considering any other possible vignettes we'd write... it's helpful to see the dplyr GitHub to see how the vignette dropdown is formatted